tests/web3js/eth_deploy_contract_and_interact_test.js

```diff
@@ -221,7 +221,7 @@ it('deploy contract and interact', async () => {
         },
         '0x1'
     )
-    assert.equal(gasEstimate, 21510n)
+    assert.equal(gasEstimate, 22026n)
```

Comment:
Change comes from: `EIP-7623: Increase calldata cost`.
See: https://mixbytes.io/blog/the-prague-electra-pectra-hardfork-explained#rec860520137 .